### PR TITLE
Add extra query parameters to search endpoints

### PR DESCRIPTION
**Pull request by JcMinarro**
Add missing query parameters when searching for Movies and TV Shows
Docs can be find on:
-. [Search Movies](https://developers.themoviedb.org/3/search/search-movies)
-. [Search TV Shows](https://developers.themoviedb.org/3/search/search-tv-shows)

### CODE REVIEW

File: tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/remote/HttpRequestBuilderExtensions.kt [hunk 1]
```diff
@@ -42,6 +42,18 @@ fun HttpRequestBuilder.parameterPage(page: Int) {
     parameter("page", page)
 }
 
+fun HttpRequestBuilder.parameterFirstAirDateYear(year: Int?) {
```
Comment (chrisnkrueger) [inline tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/remote/HttpRequestBuilderExtensions.kt:45]: The extensions are more specific for the search endpoint. 
I would move the methods into the TmdbSearchApi.

File: tmdb-api/src/jvmTest/kotlin/app/moviebase/tmdb/api/TmdbSearchApiTest.kt [hunk 2]
```diff
@@ -22,6 +25,18 @@ class TmdbSearchApiTest {
 
     val classToTest = TmdbSearchApi(client)
 
+    @Test
+    fun `it can search shows by query Simpsons on 1996`() = runBlocking {
```
Comment (chrisnkrueger) [inline tmdb-api/src/jvmTest/kotlin/app/moviebase/tmdb/api/TmdbSearchApiTest.kt:29]: The test looks good 👍

**Comment by chrisnkrueger** [comment]
Hi @JcMinarro, thank you for your contribution!

File: tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/remote/HttpRequestBuilderExtensions.kt (hunk 1 above)
Comment (JcMinarro) [reply tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/remote/HttpRequestBuilderExtensions.kt:45]: Make sense.
Let me update it

**Review by chrisnkrueger — APPROVED** [review]

**Pull request merged**